packages/ui-checkbox/src/CheckboxGroup/index.tsx

```diff
@@ -111,40 +109,46 @@ class CheckboxGroup extends Component<CheckboxGroupProps> {
   get value() {
     if (
       typeof this.props.value === 'undefined' &&
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'value' does not exist on type 'Readonly<... Remove this comment to see the full error message
       typeof this.state.value === 'undefined'
     ) {
       return []
     } else {
       return typeof this.props.value === 'undefined'
-        ? // @ts-expect-error ts-migrate(2339) FIXME: Property 'value' does not exist on type 'Readonly<... Remove this comment to see the full error message
-          [...this.state.value]
+        ? [...this.state.value]
         : [...this.props.value]
     }
   }
 
   renderChildren() {
     const { children, name, size, disabled, readOnly } = this.props
 
-    // @ts-expect-error ts-migrate(6133) FIXME: 'index' is declared but its value is never read.
-    return Children.map(children, (child, index) => {
+    return Children.map(children, (child) => {
       if (matchComponentTypes(child, [Checkbox])) {
-        return safeCloneElement(child as ReactElement, {
-          // @ts-expect-error ts-migrate(2533) FIXME: Object is possibly 'null' or 'undefined'.
-          key: `${child.props.name}`,
+        return safeCloneElement(child as React.ReactElement, {
+          key: `${
+            (child as React.ComponentElement<CheckboxProps, Checkbox>).props
+              .name
+          }`,
           name,
-          // @ts-expect-error ts-migrate(2533) FIXME: Object is possibly 'null' or 'undefined'.
-          disabled: disabled || child.props.disabled,
-          // @ts-expect-error ts-migrate(2533) FIXME: Object is possibly 'null' or 'undefined'.
-          readOnly: readOnly || child.props.readOnly,
+          disabled:
+            disabled ||
+            (child as React.ComponentElement<CheckboxProps, Checkbox>).props
+              .disabled,
+          readOnly:
+            readOnly ||
+            (child as React.ComponentElement<CheckboxProps, Checkbox>).props
+              .readOnly,
           size,
-          // @ts-expect-error ts-migrate(2533) FIXME: Object is possibly 'null' or 'undefined'.
-          checked: this.value.indexOf(child.props.value) > -1,
+          checked:
+            this.value.indexOf(
+              (child as React.ComponentElement<CheckboxProps, Checkbox>).props
+                .value!
+            ) > -1,
           onChange: this.handleChange,
-          // @ts-expect-error ts-migrate(2533) FIXME: Object is possibly 'null' or 'undefined'.
-          width: child.props.width || 'auto',
-          // @ts-expect-error ts-migrate(2339) FIXME: Property '_messagesId' does not exist on type 'Che... Remove this comment to see the full error message
-          'aria-describedby': this.hasMessages && this._messagesId
+          width:
+            (child as React.ComponentElement<CheckboxProps, Checkbox>).props
+              .width || 'auto',
+          'aria-describedby': this.hasMessages ? this._messagesId : ''
```

Comment:
If you default to `''` does it still render on the DOM element? As far as I know if you default to `undefined` it doesn't render at all, not sure about `''`.

Reply:
I'm not sure, but 'undefined' makes it clearer